rasa/utils/tensorflow/transformer.py

```diff
@@ -626,4 +626,7 @@ def call(
         # if normalization is done in encoding layers, then it should also be done
         # on the output, since the output can grow very large, being the sum of
         # a whole stack of unnormalized layer outputs.
-        return self._layer_norm(x)  # (batch_size, length, units)
+        normalized_x = self._layer_norm(x)  # (batch_size, length, units)
+
+        # apply final activation
+        return tfa.activations.gelu(normalized_x)
```

Comment:
This and the changes in diet_classifier look suspicious to me

Reply:
Yeah. We need to keep 
```
        return self._layer_norm(x)  # (batch_size, length, units)
```
and remove 
```
        normalized_x = self._layer_norm(x)  # (batch_size, length, units)

        # apply final activation
        return tfa.activations.gelu(normalized_x)
```